pg_search/tests/pg_regress/sql/mixedff_advanced_06_score_function.sql

```diff
@@ -172,8 +176,12 @@ ORDER BY a.title, a.author, a.rating, a.score, b.title;
 
 SELECT a.title, a.author, a.rating, a.score, b.title as related_title
 FROM (
-    SELECT title, author, rating, paradedb.score(id) as score
-    FROM score_test
+    SELECT title, author, rating, score
+    FROM (SELECT *, paradedb.score(id) as score
+          FROM score_test
+          WHERE content @@@ 'technology'
+          ORDER BY score DESC, title, author, rating
+          OFFSET 0) x
```

Comment:
I _think_, due to our lack of tie-breaking abilities with TopN (see #2642), that this query as originally written would output results that were largely dependent on heap insertion or indexing order.  The reason for this is that the query here (content @@@ 'technology') produces a bunch of rows that all have the same score.

So i've attempted to rewrite it a bit so that the output is consistent.

Changing how we index a table has exposed this problem, but it's been an existing problem, not a new one.

/cc @stuhood, @mdashti

Reply:
Yea, that looks right.